src/containers/Classroom.jsx

```diff
@@ -1,20 +1,27 @@
 import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
 
-import { default as ClassroomPresentational } from '../presentational/Classroom.jsx';
+import ClassroomPresentational from '../presentational/Classroom.jsx';
 import Spinner from '../presentational/Spinner.jsx';
+import { deleteClassroom } from '../actions/teacher';
 
 export default class Classroom extends Component {
 
   render() {
-    const members = this.props.classrooms.members;
-    const classroom = this.props.classrooms.data.find(classroom =>
-      classroom.id === this.props.params.classroomId);
-    if (classroom && members) {
-      return (<ClassroomPresentational data={classroom} members={members} />);
-    } else {
-      return (<Spinner />);
-    }
+    const { classrooms, params, actions } = this.props;
+    const members = classrooms.members;
+    const classroom = classrooms.data.find(classroom => classroom.id === params.classroomId);
```

Comment:
I haven't been using `.find` because [IE doesn't support it](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find). So, unless there is a babel polyfill or something that I'm not aware of, the site will break on IE.